app/Http/Middleware/CheckIfAdmin.php

```diff
@@ -28,12 +28,14 @@ class CheckIfAdmin
      */
     private function checkIfUserIsAdmin($user)
     {
-        $checkAdminUser = env('USER_IS_ADMIN', true);
+        /* $checkAdminUser = env('USER_IS_ADMIN', true);
```

Comment:
This should not be commented out.

Reply:
Forgot to uncomment it as it was needed to test before, will do it now.